### PR TITLE
GH2676 - Update PHP version for nginx prereqs

### PR DESCRIPTION
**Pull request by lorikrell**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...
Update PHP version for 2.2 and 2.3.
Created page for 2.2 based on 2.1, adding a note to the php install command.
On 2.3, update all versions from 7.0 to 7.2 and add note.
Add note regarding the version install.
Add link to system requirements.

GitHub issue: https://github.com/magento/devdocs/issues/2676

whatsnew
Update [nginx](https://devdocs.magento.com/guides/v2.2/install-gde/prereq/nginx.html) PHP versions and Composer installation steps

### CODE REVIEW

File: guides/v2.2/install-gde/prereq/nginx.md [hunk 1]
```diff
@@ -1 +0,0 @@
-../../../v2.1/install-gde/prereq/nginx.md
```
Comment (osrecio) [inline guides/v2.2/install-gde/prereq/nginx.md]: test

**Comment by jeff-matthews** [comment]
I also think we should replace the tarball download with a Composer download.

**Comment by lorikrell** [comment]
GitHub is causing weird issues with reviews on symlink replacements. Updating per your feedback, thanks!

**Comment by lorikrell** [comment]
Updated the content per @osrecio ! Thanks! Updated the versioning for 2.2 and 2.3. Also moved up the line option for composer over archive.
Working on replacing the example install steps with composer, removing the archive steps.

**Comment by lorikrell** [comment]
@jeff-matthews @osrecio  Revised content, added composer instructions (leveraged the new composer content).

File: guides/v2.1/install-gde/prereq/nginx.md [hunk 2]
```diff
@@ -75,57 +75,76 @@ Refer to [MySQL]({{ page.baseurl }}/install-gde/prereq/mysql.html) for more info
...
-*	[Clone the git repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html)
-
-For this example, we'll download and extract an archive.
+*	[Download an archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
 
-2. Change to the web server docroot directory, or to a directory you’ve configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
+*	[Clone the git repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html)
 
-		cd /var/www/html
+For this example, we'll install using Composer and the Command Line.
```
Comment (jeff-matthews) [inline guides/v2.1/install-gde/prereq/nginx.md]: "Command Line" shouldn't be capitalized. See line 87.

File: guides/v2.3/install-gde/prereq/nginx.md [hunk 3]
```diff
@@ -276,9 +298,9 @@ For this example, we'll download and extract an archive.
 
 2. Download the Magento archive, extract it, and rename the folder `magento2/`:
 
-		wget https://github.com/magento/magento2/archive/2.1.tar.gz
-		tar -xzvf 2.1.tar.gz
-		mv magento2-2.1/ magento2/
+		wget https://github.com/magento/magento2/archive/2.3.tar.gz
```
Comment (jeff-matthews) [inline guides/v2.3/install-gde/prereq/nginx.md]: REplace with Composer-based install method like 2.1.

File: guides/v2.1/install-gde/prereq/nginx.md (hunk 2 above)
Comment (lorikrell) [reply guides/v2.1/install-gde/prereq/nginx.md]: Fixed and pushed

**Comment by lorikrell** [comment]
@jeff-matthews  Also fixed that second section with the composer content under Cent OS

**Pull request merged**